velox/common/base/tests/FloatConstants.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <limits>
+
+namespace facebook::velox::test {
+
+class FloatConstants {
+ public:
+  static constexpr auto kNaNDouble = std::numeric_limits<double>::quiet_NaN();
```

Comment:
Wondering if it makes sense to shorten to kNanD, kNanF, kMaxD, kMaxF.

Reply:
Updated, thanks.